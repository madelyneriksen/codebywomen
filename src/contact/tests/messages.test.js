import React from 'react';
import renderer from 'react-test-renderer';
import Messages from '../messages.js';


describe("Messages Component", () => {
  it("Matches Snapshot On Success", () => {
    const data = {
      message: "Awesome!",
      type: "success"
    }
    const tree = renderer.create(<Messages {...data} />).toJSON();
    expect(tree).toMatchSnapshot();
  })
  it("Matches Snapshot On Error", () => {
    const data = {
      message: "Awesome!",
      type: "error"
    }
    const tree = renderer.create(<Messages {...data} />).toJSON();
    expect(tree).toMatchSnapshot();
  })
})

