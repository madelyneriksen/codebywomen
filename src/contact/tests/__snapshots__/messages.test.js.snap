// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages Component Matches Snapshot On Error 1`] = `
<div
  className="pa2"
>
  <div
    className="sans-serif pa3 ba db br3 f5 bg-washed-red b--red dark-red"
  >
    Awesome!
  </div>
</div>
`;

exports[`Messages Component Matches Snapshot On Success 1`] = `
<div
  className="pa2"
>
  <div
    className="sans-serif pa3 ba db br3 f5 bg-washed-green b--green dark-green"
  >
    Awesome!
  </div>
</div>
`;
